test-e2e/panels/ranges.test.ts

```diff
@@ -235,56 +235,11 @@ test('Range state', async ({ page }) => {
       await nmrium.page.locator('_react=MultiplicityTree').count(),
     ).toBeGreaterThan(0);
   });
-  await test.step('Change active spectrum', async () => {

```

Comment:
Why did you remove a test?

Reply:
I did remove the testing over a FID using auto-ranges picking. is it?

Reply:
Actually @jobo322 we should test that the button Auto range picking does not existing in the page anymore. You could also check that the Apodization button is there.

It is indeed weird that this test was passing because the button Auto range picking after clicking on the first spectrum should not be there anymore.